runtime-builder/build.sh

```diff
@@ -26,7 +26,7 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 debian_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
-  local digest="$(gcloud container images describe gcr.io/google-appengine/debian8:latest | \
+  local digest="$(gcloud alpha container images describe gcr.io/google-appengine/debian8:latest | \
```

Comment:
> What perhaps you may want to record in the user's image is the builder's unique tag, i.e. use BUILD_TAG env below.
Use BUILD_TAG sounds fine. Will change this to recording distroless digest.

> Do you need to use alpha here?
It got demoted to alpha in the most recent release 166.0.0